src/Sylius/Bundle/ApiBundle/Resources/config/validation/AddItemToCart.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+    <class name="Sylius\Bundle\ApiBundle\Command\Cart\AddItemToCart">
+        <property name="quantity">
+            <constraint name="NotNull" />
+            <constraint name="Positive">
+                <option name="message">sylius.cart_item.greater_than_0</option>
```

Comment:
Maybe you can use here the same message and constraint as it is in [OrderBundle](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/OrderBundle/Resources/config/validation/OrderItem.xml#L26:L30)